packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -14,9 +14,6 @@ void main() {
     List<String> listItems;
 
     void onReorder(int oldIndex, int newIndex) {
-      if (oldIndex < newIndex) {
-        newIndex -= 1;
-      }
```

Comment:
Great!

As Hans mentioned, I would suggest adding a test for:
- Moving an item to the end in a list, and asserting that the newIndex is as expected
- Moving an item to the start of a list, and asserting that the newIndex is as expected